solr/CHANGES.txt

```diff
@@ -642,6 +642,8 @@ Bug Fixes
 
 * SOLR-15558: Don't wait for zombie processes to exit when stopping. (Colvin Cowie)
 
+* SOLR-15983: Fix ClassCastException in UpdateLog$LogReplayer.doReplay. (Christine Poerschke, David Smiley)
```

Comment:
Here I've optimistically added the entry in the 9.0 bugfix section. Does that sound reasonable? The change(s) being #609 followed by #601 here.

cc: @janhoy